src/test/java/seedu/address/model/applicant/ApplicantTest.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.applicant;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANT_NAME_BENEDICT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANT_NAME_CHRIS;
+import static seedu.address.testutil.TypicalApplicants.BENEDICT;
+import static seedu.address.testutil.TypicalApplicants.CHRIS;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.ApplicantBuilder;
+
+public class ApplicantTest {
+
+    @Test
+    public void isSamePerson() {
```

Comment:
Should we change this to `isSameApplicant` instead? Since Person is gonna be deleted and `Person` is also not related to `Applicant`